codecov.yml

```diff
@@ -16,6 +16,7 @@ coverage:
         target: 90%
         flags:
           - backend
+    patch: off
```

Comment:
Patch diff status runs turned off as they are not useful compared to the main project build.